docs/API.md

```diff
@@ -581,9 +596,26 @@ An object with the following properties:
 
 #### `"error"`
 
->
-> TODO: List all possible error codes and describe what they mean enough for clients to know how react to them.
->
+* If any parameter fails to be validated an error code of -32602 is returned. The `data` properties contains an array of all error, see [Validation error data].
```

Comment:
`fails to be validated` -> `is invalid` maybe?

`data properties` -> `data property`

`all error` -> `all errors`

Reply:
Can you check this comment again? It looks to me like it was forgotten.

The same wording as this line is introduced in another place as well.

Reply:
I did forget some occurrences, my mistake